docs/codelabs/building-beautiful-sites/docsite/2-explore.md

```diff
@@ -0,0 +1,91 @@
+<!--docs:
+title: "2. Explore the starter code"
+layout: landing
+section: codelab
+path: /codelab/2-explore/
+-->
+
+<link rel="stylesheet" href="css/codelab.css" />
+
+# Explore the starter code
+
+While far from the end result we want, our skeleton site is looking pretty good already! Let’s take a look around.
+
+## Explore index.html
+
+Open up `index.html` in the starter directory. It should look like the following:
+
+```html
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
+  <title>Shrine (MDC-Web Example App)</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" sizes="192x192" href="https://material.io/static/images/simple-lp/favicons/components-192x192.png">
+  <link rel="shortcut icon" href="https://material.io/static/images/simple-lp/favicons/components-72x72.png">
+  <link rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/normalize/6.0.0/normalize.min.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet">
+  <link rel="stylesheet"
+        href="https://unpkg.com/material-components-web@0.9.1/dist/material-components-web.min.css">
+  <link rel="stylesheet" href="app.css">
+</head>
+<body class="mdc-typography">
+  <h1 class="mdc-typography--display4">Shrine</h1>
+  <main>
+    <div class="mdc-card shrine-product-card">
+      <section class="mdc-card__primary">
+        <span class="mdc-card__title shrine-product-card__price">$20</span>
+      </section>
+      <img class="shrine-product-card__image" width="240" height="240" alt="Sunglasses" src="assets/sunnies.png">
+    </div>
+    <!-- ... -->
+```
+
+The first thing to notice here is that we’re using a `<link>` tag to load a fully-built version of MDC-Web via the [unpkg CDN](https://unpkg.com/#/). We publish MDC-Web via NPM. Using unpkg is a great way to experiment with MDC-Web without the need to immediately include it as a dependency. Note that we include [normalize.css](https://necolas.github.io/normalize.css/) for consistent cross-browser rendering, as well as the [Roboto font](https://fonts.google.com/specimen/Roboto) from Google Fonts.
```

Comment:
Consider writing: 
Note that we pinned to use version `0.9.1` to the purpose of this codelab as the `latest` version is under active development. For the available components and latest APIs, please checkout our [Github Repository](https://github.com/material-components/material-components-web) and [Demo site](http://material-components-web.appspot.com/).